subprojects/performance/src/performanceTest/groovy/org/gradle/performance/regression/android/RealLifeAndroidBuildPerformanceTest.groovy

```diff
@@ -45,11 +47,18 @@ class RealLifeAndroidBuildPerformanceTest extends AbstractCrossVersionGradleProf
         given:
         testProject.configure(runner)
         runner.tasksToRun = tasks.split(' ')
-        runner.args = parallel ? ['-Dorg.gradle.parallel=true'] : []
+        if (parallel) {
+            runner.args.add('-Dorg.gradle.parallel=true')
+        }
         runner.warmUpRuns = warmUpRuns
         runner.runs = runs
         applyEnterprisePlugin()
 
+        and:
+        if (testProject == SANTA_TRACKER_KOTLIN) {
```

Comment:
Why don't we call the method everytime? I think the `configureForLatestAgpVersionOfMinor` method should be on `AndroidTestProject`.

Reply:
Because it doesn't apply to K9 or `perf-large-android`.

Reply:
It doesn't have any effect. Though the method could still be called either with an empty implementation or passing the system property.

I am also fine with leaving as is.

Reply:
I'll leave it as is because seeing it called without having any effect is confusing to me.